.env.example

```diff
@@ -5,7 +5,7 @@ BOT_ADMINS=123,456
 AUTOROLE=MSG_ID:ROLE_ID:EMOJI:AUTOREMOVE
 # Another example: AUTOROLE=738932146978160661:728202487672078368:❌:false,738932146978160661:738936540465725520:✅:true
 
-DATABASE_URL="localhost:5432/tsc-bot"
+DATABASE_URL="postgres://tscbot:tscbot@localhost:5432/tscbot"
```

Comment:
The provided example has no hope of working, this is what I had to set it to to connect to the docker created DB